### ISSUE TITLE
Project support suspended

### ISSUE THREAD

**Issue by grafov** [grafov/m3u8#225]
Project support suspended for the time. You are free to fork and evolve as you wish!

TL;DR: Unfortunately, I no longer have time to maintain this project. HLS is out of scope in my current work. Years ago, the code was important for video streaming projects I worked on, and it may be still used in some proprietary production code (at least one project as I know). However, this could pose risks, as I no longer monitor updates to the HLS standard or address potential security vulnerabilities.

I want to express my heartfelt thanks to all the volunteers who contributed to this project. Special thanks to those who have maintained the code for so long. Your contributions are forever recorded in the commit history …well, at least until GitHub decides to drop this archived piece of code :) Regardless, this was an interesting and worthwhile endeavor.

Sadly, life is short, and scope of possible activities is quite limited. I'll dedicate my time to other projects in FOSS.
